TransformationPlayground/playground/transform_expert/prompting/knowledge/os_2_17.py

```diff
@@ -0,0 +1,453 @@
+INDEX_GUIDANCE = """"""
+
+INDEX_KNOWLEDGE = """
```

Comment:
I'm sure we'll iterate on the ideas here, but - I see "guidance" as the hints we provide the LLM when there are multiple valid choices it could pursue (separate indices vs union index when resolving multi-type mappings, for example).  The end user will be able to override this guidance with their own directives via the Playground UI, as well.

Knowledge is stuff that is fundamentally true (from the LLM's perspective) about the task at hand and unlikely to change.  This is stuff like the API spec for a specific version of the source/target.